modules/indexing/src/test/java/org/apache/ignite/util/GridCommandHandlerIndexingTest.java

```diff
@@ -105,9 +107,33 @@ public void testCorruptedIndexPartitionShouldFailValidation() throws Exception {
 
         injectTestSystemOut();
 
+        assertEquals(EXIT_CODE_OK, execute("--cache", "validate_indexes", "--check-crc", CACHE_NAME));
+
+        assertContains(log, testOut.toString(), "issues found (listed above)");
+    }
+
+    /**
+     * Tests with that corrupted pages in the index partition are detected.
+     */
+    @Test
+    public void testCorruptedIndexPartitionShouldFailValidationWithoutCrc() throws Exception {
+        Ignite ignite = prepareGridForTest();
+
+        forceCheckpoint();
+
+        stopAllGrids();
+
+        FileUtils.copyFile(new File(getClass().getResource("index.bin").toURI()), indexPartition(ignite, GROUP_NAME));
+        startGrids(2);
```

Comment:
why u start 2 grids here? may be would more clear to publish "2" from prepareGridForTest internally?

Reply:
I've added a constant to the test class to store nodes counter.